WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -48,5 +49,27 @@ private static void ModifyRegistry(bool runOnSystemStartup, string pathToExeFile
 				key.DeleteValue(nameof(WalletWasabi), false);
 			}
 		}
+
+		private static void ModifyMacOsLoginItems(bool runOnSystemStartup)
+		{
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+			{
+				throw new InvalidOperationException("Running osascript can only be done on MacOS.");
+			}
+
+			string argumentToAddWasabiToMacOsStartupSetting = $"-c \"osascript -e \' tell application \\\"System Events\\\" to make new login item at end of login items with properties {{name:\\\"{nameof(WalletWasabi)}\\\", path:\\\"/Applications/WasabiWallet.app\\\",hidden:false}} \' \"";
+			string argumentToDeleteWasabiFromMacOsStartupSetting = $"-c \"osascript -e \' tell application \\\"System Events\\\" to delete login item \\\"{nameof(WalletWasabi)}\\\" \' \"";
+
+			ProcessStartInfo processInfo = new()
+			{
+				UseShellExecute = true,
+				WindowStyle = ProcessWindowStyle.Normal,
```

Comment:
Why `Normal`? I would expect rather `Hidden`.

Reply:
Strange because it didn't open any window for me. But you are right, we don't need to open any window for this.